docs/advtool-install

```diff
@@ -151,7 +151,9 @@ function myFunction() {
 </div>
 
 <div class="ibm-col-12-9 ibm-padding-top-1 ibm-padding-bottom-1">
-<table class="ibm-data-table ibm-grid" data-widget="datatable" id="distro-table">
+<div id="distro-table"></div>
+
+<table class="bx--data-table">
```

Comment:
Could you add this change to the commit message?

Reply:
Done